docs/standard/modernize-with-azure-and-containers/modernize-existing-apps-to-cloud-optimized/deploy-existing-net-apps-as-windows-containers.md

```diff
@@ -129,6 +129,37 @@ Running a container on Windows with or without Hyper-V isolation is a run-time d
 
     [https://info.microsoft.com/rs/157-GQE-382/images/Container%20infographic%201.4.17.pdf](https://info.microsoft.com/rs/157-GQE-382/images/Container%20infographic%201.4.17.pdf)
 
+
+## The container ecosystem in Azure
+
+In previous sections, it’s been explained what the benefits of Docker containers are as well as details on the specific container images for .NET applications. All that generic information is fundamental in order to develop or containerize an application.
+However, when thinking about the production deployment environment or even QA and Dev/Test environments, Microsoft Azure provides an open and broad variety of choices, a full container ecosystem in the cloud (shown in the diagram below). Depending on your specific application’s needs, you should choose one or another Azure product.
+
+![](./media/image7.5.png)
+
+> **Figure 4-7.5.** The container ecosystem in Azure
+
+From the container ecosystem in Azure, the following products supporting containers that are considered infrastructure:
+-   **Azure Container Instances (ACI)**
+-   **Azure Virtual Machines** (With container's support)
+-   **Azure Virtual Machine Scale Sets** (With container's support)
+
+From those three, ACI provides a great benefit, which is the fact that you don't need to maintain the underlaying OS, no need for you to upgrade/path, etc. but still ACI is positioned in the infrastructure level, as better explained in the upcoming sections of this document.
```

Comment:
underlaying  -> underlying 

upgrade/path -> upgrade/patch?

but still ACI is -> But ACI is still

this document -> chapter or book?